gobblin-runtime/src/main/java/org/apache/gobblin/runtime/TaskConfigurationKeys.java

```diff
@@ -35,7 +35,11 @@ public class TaskConfigurationKeys {
   public static final String TASK_IS_SINGLE_BRANCH_SYNCHRONOUS = "gobblin.task.is.single.branch.synchronous";
   public static final String DEFAULT_TASK_IS_SINGLE_BRANCH_SYNCHRONOUS = Boolean.toString(false);
 
-
   public static final String TASK_SKIP_ERROR_RECORDS = "task.skip.error.records";
   public static final long DEFAULT_TASK_SKIP_ERROR_RECORDS = 0;
+
+  public static final String TASK_DISABLE_FAILED_EVENTS = "gobblin.task.failure.disable.event.emission";
```

Comment:
*nit
It should be "gobblin.task.disable.failed.event.emission" to match the string name